Orchestrator.py

```diff
@@ -140,17 +144,137 @@ def receive(self,frame):
             duration = frame['movementDuration'],
         )
 
+        # update explored cells
+        self.obstacleCells  += [self._xy2hCell(newX, newY)]
+        cellsTraversed       = self._computeCellsTraversed(dotbot['x'], dotbot['y'], newX, newY)
+        self.exploredCells  += [c for c in cellsTraversed if c not in self.obstacleCells]
```

Comment:
I don't understand why `if c not in self.obstacleCells`, seems you sweep a bug under the rug. If a DotBot has gone from A to B, there are no obstacles in-between.